src/admin/class-sign-in-with-google-admin.php

```diff
@@ -695,7 +695,13 @@ protected function set_access_token( $code = '' ) {
 			),
 		);
 
+		// handle wp_remote_* returning a \WP_Error
 		$response = wp_remote_post( 'https://www.googleapis.com/oauth2/v4/token', $args );
+		if ( is_wp_error( $response ) ) {
+			$message = static::class . ": Remote post to fetch token failed. " . $response->get_error_code() . ": " . $response->get_error_message();
+			error_log( $message );
+			return false;
```

Comment:
The method doc block states that the return is either an Access Token or a `WP_Error`. Would it be better to return the `WP_Error` to keep the method documentation accurate and handle the error where this is called?